packages/shared/src/utils/get-simple-hash.test.ts

```diff
@@ -0,0 +1,11 @@
+import { getSimpleHash } from './get-simple-hash';
+
+describe('getSimpleHash', () => {
+	it('returns "364492" for string "test"', () => {
+		expect(getSimpleHash('test')).toBe('364492');
+	});
+
+	it('returns "28cb67ba" for stringified object "{ key: \'value\' }"', () => {
+		expect(getSimpleHash(JSON.stringify({ key: 'value' }))).toBe('28cb67ba');
+	});
+});
```

Comment:
@jaycammarano hope I didn't butcher this, but do let me know if there's better ways to go about testing this hash function other than having a list of before/after values 😅

Reply:
I don't think there's another way! Unit tests make sure the output of a function is correct based on the known input, which is exactly what you're doing here 🙂